### PR TITLE
RWA-3601: Add logging

### PR DESCRIPTION
**Pull request by SammiChong**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-3601

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java [hunk 1]
```diff
@@ -120,6 +121,7 @@ public GetTasksCompletableResponse<Task> searchForCompletableTasks(
         List<String> taskTypes = extractTaskTypes(evaluateDmnResult);
 
         if (taskTypes.isEmpty()) {
+            log.info("TaskTypes is empty");
             return new GetTasksCompletableResponse<>(false, emptyList());
         }
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java:126]: There are also some logging improvements/additions for the `TaskSearchController.searchWithCriteriaForAutomaticCompletion` .  E.g. when no role assignments are found output the userId.  We could also output the caseId and eventId for every request.

**Review by raghera — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java (hunk 1 above)
Comment (SammiChong) [reply src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java:128]: UserId is already logged in https://github.com/hmcts/wa-task-management-api/blob/6b290b8fc5cd2a75d26f171ca967c9272be32b04/src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/access/AccessControlService.java#L37 which is called in https://github.com/hmcts/wa-task-management-api/blob/22c70d10d6c0e7732da77f271e0f2f25bd0d1f6b/src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskSearchController.java#L110

`caseId` is already logged as well but I've added in `eventId`

Comment (raghera) [reply src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java:128]: Logs updated in the Controller class so resolved.

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskSearchController.java [hunk 2]
```diff
@@ -196,9 +198,8 @@ public ResponseEntity<GetTasksCompletableResponse<Task>> searchWithCriteriaForAu
             permissionsRequired
         );
 
-        log.info(String.format("POST /search-for-completable - userId: %s, caseId: %s, taskIds: %s",
+        log.info(String.format("POST /search-for-completable - userId: %s, taskIds: %s",
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskSearchController.java:201]: Let's keep the caseId here for now.

**Review by raghera — APPROVED** [review]

**Review by skandala-cgi — APPROVED** [review]

**Pull request merged**